figs/species_fig.R

```diff
@@ -18,11 +17,12 @@ species<-tibble(species=tree$tip.label) %>%
                                           "Streptococcus agalactiae", "Drosophila sechellia", "Papio anubis",
                                           "Anolis carolinensis", "Gasterosteus aculeatus"), "hackathon", "new")))
 
+pdf(file="species_fig.pdf", width=6, height=8, pointsize=10)
```

Comment:
We're making a pdf from latex, so want a pdf (ie something with vector graphics).